source/_layouts/page-pattern.html

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+---
+<div id="main" class="col-md-12 clearfix" role="main">
+  <article class="post">
+    <header class="page-tabs-header">
+      <div class="row">
+        <div class="col-sm-12 col-md-6 cols-nopad-right">
+          <h1>{{ page.title }}</h1>
+        </div>
+        <div class="col-sm-12 col-md-6 cols-nopad-left cols-nopad-right">
+          {% if page.code == false %}
+            {% include nav-tabs-pattern.html codetab=false %}
+          {% elsif page.design == false %}
+            {% include nav-tabs-pattern.html designtab=false %}
+          {% else %}
+            {% include nav-tabs-pattern.html %}
+          {% endif %}
+        </div>
+      </div>
+    </header>
+    <section class="post_content">
+      <div class="tab-content">
+        <div role="tabpanel" class="tab-pane active" id="overview">
+          {% capture my_include %}{% include {{page.overview}} %}{% endcapture %}
+          {{ my_include | site_replace | markdownify }}
+        </div>
+        <div role="tabpanel" class="tab-pane" id="design">
+          {% capture my_include %}{% include {{page.design}} %}{% endcapture %}
+          {{ my_include | site_replace | markdownify }}
+        </div>
+        <div role="tabpanel" class="tab-pane" id="code">
```

Comment:
It is more the case that the code tab displays multiple examples. Typically, there is only one angular example, but many html/css (core) examples are shown in that tab.